Client/Frontend/Browser/Tabs/State/TabTrayState.swift

```diff
@@ -5,26 +5,31 @@
 import Foundation
 
 struct TabTrayState {
```

Comment:
Wondering what could be tested in here, some is mock and doesn't have real data, but wondering if there's anythign we could do so far

Reply:
I was leaving the Unit test for the action tickets because I will have a better picture then if anything is missing from the state and the flow